### DISCUSSION TITLE
Subtitle download from opensubtitles.com instead from .org

### DISCUSSION THREAD

**Discussion by StvVl** [smplayer-dev/smplayer#1044] [Ideas]
As it would appear that opensubtitles.org requires a VIP (paid) membership to download subtitles via API, and that opensubtitles.com allows downloading subtitles for free, I am wondering if it would be possible to add an option to downolad subtitles from  opensubtitles.com.
PS: When trying to download subtitles from opensubtitles.org, it will look as if it is working, but every subtitle you dovnload will be empty except the message that VIP is required to download subtitles.

**Comment by smplayer-dev** [comment]
What version of smplayer are you using?
I think this was already fixed a long time ago.

**Reply by StvVl** [reply]
It would appear that I was using an older version (22.something). Sorry, I forgot that the player doesnt auto-update on windows. It works fine on the newest version.
